### ISSUE TITLE
Support: CREATE INDEX IF NOT EXISTS

### ISSUE THREAD

**Issue by cyrille-arundo** [oguimbal/pg-mem#66]
```sql¨
CREATE TABLE my_table (
    id text NOT NULL PRIMARY KEY,
    field1 text,
    field2 text NOT NULL,
    parent_id text
);

CREATE INDEX IF NOT EXISTS my_table_id_parent ON my_table (id, parent_id);'


🔨 Not supported 🔨 : The query you ran generated an AST which parts have not been read by the query planner. This means that those parts could be ignored:

    ⇨ .ifNotExists (true)

👉 pg-mem is work-in-progress, and it would seem that you've hit one of its limits.

*️⃣ Reconsituted failed SQL statement: CREATE INDEX  IF NOT EXISTS "my_table_id_parent" ON "my_table" ("id" , "parent_id" )
 ```

**Comment by oguimbal** [comment]
Should be fixed in `pg-mem@1.7.6`

**Comment by cyrille-arundo** [comment]
Thanks for the fix @oguimbal. I verified it and it works as expected in our unit-tests.
